File: atcoder/atcoder_problems/abc001~100/abc051~100/abc088/abc088_b.py
N = int(input())
A = list(map(int, input().split()))

A.sort(reverse=True)

alice = 0
bob = 0

for i in range(N):
    if i % 2 == 0:
        alice += A[i]
    else:
        bob += A[i]

print(alice - bob)




# N = int(input())
# A = list(map(int, input().split()))

# a = 0
# b = 0

# for i in range(N):
#     m = max(A)
#     if i % 2 == 0:
#         a += m
#     else:
#         b += m
#     A.remove(m) 

# print(a - b)

File: atcoder/atcoder_problems/abc001~100/abc051~100/abc088/abc088_b.cpp
#include <iostream>
#include <vector>
#include <algorithm>
using namespace std;


int main(){
    int N, A[101];
    cin >> N;
    for (int i = 0; i < N; i++) cin >> A[i];
    sort(A, A + N, greater<int>());

    int v[2] = {0 , 0};
    for (int i = 0; i < N; i++){
        v[i % 2] += A[i];
    }
    int ans = v[0] - v[1];
    cout << ans << endl;
    return 0;
}





// int main (){
//     int N,a;
//     cin >> N;
//     vector<int> A;
//     for (int i = 0; i < N; i++){
//         cin >> a;
//         A.push_back(a);
//     };
//     sort(A.begin(), A.end());
//     reverse(A.begin(), A.end());

//     int Alice = 0, Bob = 0;
//     for (int i = 0; i < N; i++){
//         if (i % 2 == 0) Alice += A[i];
//         else Bob += A[i];
//     }
//     cout << Alice - Bob << endl;
//     return 0;
// }